eftcamb/06_abstract_EFTCAMB_model.f90

```diff
@@ -78,6 +78,7 @@ procedure :: compute_rhoQPQ              => EFTCAMBModelComputeRhoQPQ
         procedure :: compute_Einstein_factors    => EFTCAMBModelComputeEinsteinFactors                    !< subroutine that computes the Einstein equations factors. For details refer to the numerical notes.
         procedure :: compute_pi_factors          => EFTCAMBModelComputePiFactors                          !< subroutine that computes the pi field equations factors. For details refer to the numerical notes.
         procedure :: compute_tensor_factors      => EFTCAMBModelComputeTensorFactors                      !< subroutine that computes the factors for the tensor propagation equation. For details refer to the numerical notes.
+        procedure :: compute_kinetic_gradient    => EFTCAMBModelComputeKineticGradient                    !< subroutine that computes the kinetic and gradient terms. For details refer to the numerical notes.
```

Comment:
Would make the name less dependent on the actual implementation of the stability conditions.
Something like compute_stability_quantities or similar.